server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/OscService.java

```diff
@@ -102,6 +115,17 @@ public OscSwapTableVO swapTable(@PathVariable Long scheduleTaskId) {
         OnlineSchemaChangeParameters oscParameters = JsonUtils.fromJson(scheduleEntity.get().getJobParametersJson(),
                 OnlineSchemaChangeParameters.class);
 
+        // check permission, only creator can swap table manual
+        PreConditions.validHasPermission(
+                Objects.equals(authenticationFacade.currentUserId(), scheduleEntity.get().getCreatorId()),
+                ErrorCodes.AccessDenied,
+                "no permission swap table name, schedule task id " + scheduleTaskId);
```

Comment:
error message should be readble, user may not undertand schedule task id here.

Reply:
okey, done